### PR TITLE
CB-15909 Change rootVolume to 100GB for all types of nodes. Updated r…

### PR DESCRIPTION
**Pull request by vbrodetskyi**
…ootVolume from 50 GB to 100 GB for Yarn to unblock tests. If rootVolume lower than 100 GB then cluster deploy fails.

See detailed description in the commit message.

### CODE REVIEW

**Comment by vbrodetskyi** [comment]
@lnardai @doktoric we need this change for 7.2.14, could you please review it?

**Comment by lnardai** [comment]
Can you link any details what changed? Why do we need to double the volumeSize?

**Review by lajosrodek — CHANGES_REQUESTED** [review]
Changes look okay. Could you please apply them to the 7.2.15 cluster definitions as well?

**Comment by vbrodetskyi** [comment]
> Can you link any details what changed? Why do we need to double the volumeSize? Does this needed for prod? Why can't this be targeted to master?

@lnardai @lajosrodek we need that because cluster deploy fails in case rootVolume size is lower than 100GB. For AWS, GCP and Azure this issue is already fixed, so i decided to fix it for YCloud too. I will also commit that to master in separate PR for 7.2.14 and 7.2.15.

**Comment by lnardai** [comment]
ycloud is only supported internally. If this is needed for mow-int, might worth to consider CB-2.52.0 as it's the next release branch.
"we need that because cluster deploy fails in case rootVolume size is lower than 100GB." - ok, so, let me rephrase the question. Have you checked what takes up so much additional space that we need to double these values? How much do we actually use of the extra 50GB?
